lbrynet/core/Wallet.py

```diff
@@ -313,12 +314,12 @@ def get_txid_for_name(self, name):
         d.addCallback(lambda r: None if 'txid' not in r else r['txid'])
         return d
 
-    def get_stream_info_from_txid(self, name, txid):
+    def get_stream_info_from_txid(self, name, txid, nout):
         d = self.get_claims_from_tx(txid)
 
         def get_claim_for_name(claims):
             for claim in claims:
-                if claim['name'] == name:
+                if claim['name'] == name and claim['nOut'] == nout:
```

Comment:
This is a good idea, but I'm gonna hold this off as the last thing to do, since its going to change the interface to a bunch of functions and will take some time to code. If I don't have time for it this week, it will have to go into next release cycle.